### PR TITLE
FireEyeHX enhancments

### PR DESCRIPTION
**Pull request by dorschw**
fixes: https://github.com/demisto/etc/issues/47380

adds the following commands

-  fireeye-hx-delete-condition
-  fireeye-hx-delete-indicator
-  fireeye-hx-list-indicator-category

### CODE REVIEW

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml [hunk 1]
```diff
@@ -1589,12 +1589,119 @@ script:
...
     description: Delete data acquisition.
     name: fireeye-hx-delete-data-acquisition
+  - name: fireeye-hx-delete-indicator-condition
+    arguments:
+    - name: category
+      required: true
+      description: The indicator category.
+    - name: indicator_name
+      required: true
+      description: The indicator name. The name of the indicator. Use the `uri_name` value.
```
Comment (DeanArbel) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml:1599]: The indicator name? or The name of the indicator?

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py [hunk 2]
```diff
@@ -2295,20 +2315,19 @@ def suppress_alert_command(client: Client, args: Dict[str, Any]) -> CommandResul
...
-    if args.get('alerted'):
-        args['alerted'] = args['alerted'] == 'yes'
-    if args.get('sort'):
-        args['sort'] = sort_map.get(args.get('sort', ''))
+    if limit := args.get('limit'):
+        args['limit'] = int(limit)
+    if alerted := args.get('alerted'):
+        args['alerted'] = alerted == 'yes'
+    if sort := args.get('sort'):
+        args['sort'] = sort_map.get(sort)
```
Comment (DeanArbel) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py:2330]: 🥇

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py [hunk 3]
```diff
@@ -2397,6 +2489,36 @@ def append_conditions_command(client: Client, args: Dict[str, Any]) -> CommandRe
     )
 
 
+def delete_condition_command(client: Client, args: Dict[str, str]) -> CommandResults:
+    # XSOAR yml makes sure the args exist
```
Comment (DeanArbel) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py:2493]:
```suggestion
    # Mandatory args - always exist
```

File: Packs/FireEyeHX/pack_metadata.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
     "name": "FireEye HX",
     "description": "FireEye Endpoint Security is an integrated solution that detects what others miss and protects endpoint against known and unknown threats. The FireEye HX Demisto integration provides access to information about endpoints, acquisitions, alerts, indicators, and containment. Customers can extract critical data and effectively operate security operations automated playbooks.",
     "support": "xsoar",
-    "currentVersion": "2.0.0",
+    "currentVersion": "2.0.1",
```
Comment (DeanArbel) [inline Packs/FireEyeHX/pack_metadata.json:5]: You can give this a minor version, it is new commands.

**Review by DeanArbel — APPROVED** [review]
Good job, keep updating older codes.

**Comment by DeanArbel** [comment]
@jrosenpanw Please assign someone from your team to review the `yml` and `releasenotes` changes.

@dorschw Please make sure to add to the `README.md` the new commands after the technical writer review.

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml (hunk 1 above)
Comment (dorschw) [reply Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml:1599]: image

**Comment by ShirleyDenkberg** [comment]
@dorschw New commands need to be added to the integration README.
@DeanArbel Doc review completed.

**Pull request merged**